lib/LaTeXML/Package.pm

```diff
@@ -1033,7 +1041,8 @@ sub DefMacroI {
     $STATE->assignMathcode($cs => 0x8000, $options{scope}); }
   $cs        = coerceCS($cs);
   $paramlist = parseParameters($paramlist, $cs) if defined $paramlist && !ref $paramlist;
-  $STATE->installDefinition(LaTeXML::Core::Definition::Expandable->new($cs, $paramlist, $expansion, %options),
+  my $defcs = ($options{robust} ? defRobustCS($cs, $options{scope}) : $cs);
+  $STATE->installDefinition(LaTeXML::Core::Definition::Expandable->new($defcs, $paramlist, $expansion, %options),
     $options{scope});
   AssignValue(ToString($cs) . ":locked" => 1, 'global') if $options{locked};
```

Comment:
should we also lock `$defcs` here - when available - or is that too much safety for no benefit?